superset-frontend/src/components/ImportModal/index.tsx

```diff
@@ -261,11 +291,96 @@ const ImportModelsModal: FunctionComponent<ImportModelsModalProps> = ({
     );
   };
 
+  const isAnyOverwriteFieldSelected = (): boolean =>
+    Object.values(configOverwriteStatus).some(status => status);
+
+  const confirmOverwriteConfigByTyping = (): JSX.Element => {
+    const expectedInput = `${t('CONFIRM')}`;
+
+    return (
+      <StyledInputContainer>
+        <div className="control-label">
+          {t(
+            `Type "${expectedInput}" to confirm loading the config, ` +
+              `otherwise the configuration will not loaded`,
```

Comment:
```suggestion
            'Type "%(expectedInput)s" to confirm loading the config, ' +
              'otherwise the configuration will not loaded', expectedInput
```

Reply:
Done